Advance-2/index.php

```diff
@@ -0,0 +1,54 @@
+<?php
+
+    use PHPMailer\PHPMailer\PHPMailer;
+    require 'vendor/autoload.php';
+
+    if(isset($_POST['submit'])) {
+        $email = $_POST['email'];
+        $mail = new PHPMailer(true);                      
+        $mail -> isSMTP();                          
+        $mail -> Host = 'smtp.gmail.com';
+        $mail -> SMTPAuth = true;                      
+        $mail -> Username = 'sandip.poddar@innoraft.com';             
+        $mail -> Password = 'pbhcctzzenyutbsj';                       
```

Comment:
Explore about `$_ENV` global variable. Add these creds in env file and fetch the values from it.